### PR TITLE
feat: Emotion native

### PR DESCRIPTION
**Pull request by lalrg**

### CODE REVIEW

**Comment by kevinrodriguez-io** [comment]
@gaboesquivel Let's ask wolf for some insight here

File: src/components/BottomButton/index.tsx [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
+  })
+
+  const Text = styled.Text({
     textAlign: 'center',
     fontFamily: 'Montserrat-Regular',
     color: colors.flatWhite.light,
     textTransform: 'uppercase'
-  },
-})
+  })
```
[the remark this answers is not in the document]
Comment (gaboesquivel) [reply src/components/BottomButton/index.tsx]: Yeah, good catch! we should define them outside the component function.

[the remark this answers is not in the document]
Comment (lalrg) [reply src/components/BottomButton/index.tsx]: Solved!

**Comment by kevinrodriguez-io** [comment]
@lalrg 
I added some changes to this:
- Added v11 again
- Added types for TS support
- Tested
- Corrected some code issues
- Everything is looking good

**Pull request merged**